backend/data/processors/conditions_preprocessing.py

```diff
@@ -432,6 +442,134 @@ def map_word_to_program_type(processed, regex_word, type):
         flags=re.IGNORECASE,
     )  # hard to capture a generic case?
 
+#adds handbook notes and removes phrasing
+def remove_extraneous_comm_data(conditions, processed):
+    note = ""
+    # Academic Standing note
+    processed, number_of_subs = re.subn(
+        r"Prerequisite: Students must be in Good Academic Standing<br/><br/>", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Students must be in Good Academic Standing."
+
+    processed, number_of_subs = re.subn(
+        r"on Good Standing and", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Students must be in Good Academic Standing."
+
+    processed, number_of_subs = re.subn(
+        r"Good Standing", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Students must be in Good Academic Standing."
+    
+
+    # Final 2 terms note
+    processed, number_of_subs = re.subn(
+        r"It is strongly recommended that students only complete this course within their final 2 terms of study", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Recommended to take this course within their final 2 terms of study."
+
+    # Business school students notes
+    processed, number_of_subs = re.subn(
+        r"Only available to single and double degree Business School students in Term 1. It will be offered to non-Business School students in Terms 2 and 3.<br/><br/>", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Only available to single and double degree Business School students in Term 1. It will be offered to non-Business School students in Terms 2 and 3."
+    
+    processed, number_of_subs = re.subn(
+    r"Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Terms 1 and 3.<br/><br/>", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Terms 1 and 3."
+    
+    processed, number_of_subs = re.subn(
+    r"Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3.<br/><br/>", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3.<br/><br/>"
```

Comment:
Make sure you take the HTML out of the handbook note
edit: look at peter's comment about the same thing